contrib/hack/test-apiserver.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
+export PATH=${ROOT}/contrib/hack:${PATH}
+
+start-server.sh
+
+# create a few resources
+kubectl create -f contrib/examples/apiserver/broker.yaml
```

Comment:
our kubectl script should be the same as just running the normal kubectl binary. If people need more/better/different error codes or messages then that's something outside of this script to do.